c.md

```diff
@@ -9,7 +9,23 @@ updated: 2020-04-25
 
 * scanf("%d", &x) -- read value into the variable x from input stream
 * Printf("%d",x) -- printf value to the output stream
-* gets(str) -- reads a line from input stream into a variable
+* gets(str) -- reads a line from input stream into a variable\
+
+## Header Files
```

Comment:
move this to bottom of the cheat sheet

Reply:
Moved this to the bottom of the cheat sheet. Please verify and let me know if any changes are required.